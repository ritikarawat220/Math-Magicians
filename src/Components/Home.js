import React from 'react';
import style from './Home.module.css';

const Home = () => (
  <div className={style.contentContainer}>
    <h1>Welcome to our page!</h1>
    <div className={style.paragraphs}>
      <p>
        A calculator is a handy tool for quickly performing basic mathematical
        calculations. It can save time and effort compared to doing calculations
        by hand and also reduce the potential for errors. Basic calculators can
        perform basic mathematical operations such as addition, subtraction,
        multiplication and division. They are widely used in many fields such as
        everyday life, business, and education.

        A basic calculator is not just a tool for performing mathematical
        calculations, it is also a tool for simplifying everyday life.
        It can help with budgeting and managing finances, assist with
        shopping and comparing prices, and can even be used for cooking
        and recipe conversions.
      </p>
      <p>
        It is a device that can be found in nearly every household and office,
        and it has become a necessary accessory for many people. With the advancement
        of technology, many calculators now have additional features such as memory and
        percentage functions, making them even more versatile and useful.
        It&apos;s not just a tool but a companion which can make your day to day
        calculation easier and accurate.
      </p>
    </div>
  </div>
);

export default Home;
