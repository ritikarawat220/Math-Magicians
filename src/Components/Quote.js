import React, { useState, useEffect } from 'react';
import style from './Quote.module.css';

function Quote() {
  const category = 'life';
  const key = 'ojAjYBdnDJ1mTn7+NDqEFA==UUXIGSlNnsU0RurB';

  const [data, setData] = useState([]);
  // const [hasError, setHasError] = useState(null);
  const [isLoading, setIsLoading] = useState(true);

  const setHasError = null;

  async function fetchData() {
    try {
      const response = await fetch(
        `https://api.api-ninjas.com/v1/quotes?category=${category}`,
        {
          headers: {
            'x-api-key': key,
          },
        },
      );
      const jsonData = await response.json();
      setData(jsonData);
      setIsLoading(false);
    } catch (error) {
      setHasError(error);
      setIsLoading(false);
    }
  }

  useEffect(() => {
    fetchData();
  }, []);

  return (
    <main>
      <div className={style.contentContainer}>
        {isLoading ? (
          <p>Loading...</p>
        ) : (
          <ul>
            {data.map((item) => (
              <li key={item.quote}>{item.quote}</li>
            ))}
          </ul>
        )}
      </div>
    </main>
  );
}

export default Quote;
