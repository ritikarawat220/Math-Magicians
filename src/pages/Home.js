import React from 'react';
import styles from './Home.module.css';

const Home = () => (
  <div className={styles.home}>
    <h2 className={styles.headline}>
      Welcome to our page!
    </h2>
    <p>
      A calculator is a useful tool for performing basic
      mathematical operations quickly and easily.
      It is a small, handheld device that can be found in
      nearly every household and office.
      Calculators can perform basic arithmetic functions like
      addition, subtraction, multiplication, and division,
      as well as more advanced functions
      like percentages and memory storage.
      In addition to being a tool for performing calculations,
      a calculator can simplify everyday life in many ways.
      It can be used for budgeting and managing finances,
      helping with shopping and comparing prices, and even
      for cooking and recipe conversions.
    </p>
    <p>
      With the advancement of technology, many calculators now have additional
      features that make them even more versatile and useful.
      Some calculators have large displays for easy reading,
      while others are designed to be lightweight and portable.
      There are also calculators that can perform complex scientific
      and engineering calculations.
      In short, a calculator is not just a tool but a companion
      that can make your day-to-day calculations easier and more accurate.
      It has become a necessary accessory for many people in fields
      such as everyday life, business, and education.
    </p>
  </div>
);
export default Home;
