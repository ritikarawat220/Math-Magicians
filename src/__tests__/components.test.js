import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import React from 'react';
import Quote from '../Components/Quote';
import Home from '../Components/Home';
import Calculator from '../Components/calculator';

describe('Test components', () => {
  test('Snapshot for Home ', () => {
    const tree = render(<Home />);
    expect(tree).toMatchSnapshot();
  });

  test('Snapshot for Quote', () => {
    const tree = render(<Quote />);
    expect(tree).toMatchSnapshot();
  });

  test('Snapshot for Calculator', () => {
    const tree = render(<Calculator />);
    expect(tree).toMatchSnapshot();
  });
});
