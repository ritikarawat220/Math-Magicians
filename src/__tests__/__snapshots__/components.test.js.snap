// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test components Calculator snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="contentContainer"
      >
        <h2>
          Let's do some calculations!
        </h2>
        <div
          class="calculator-body"
        >
          <div
            class="screen"
            data-testid="result"
          >
            0
          </div>
          <div
            class="keyboard"
          >
            <div
              class="row"
            >
              <button
                class="btn"
                type="button"
              >
                AC
              </button>
              <button
                class="btn"
                type="button"
              >
                +/-
              </button>
              <button
                class="btn"
                type="button"
              >
                %
              </button>
              <button
                class="btn"
                type="button"
              >
                รท
              </button>
            </div>
            <div
              class="row"
            >
              <button
                class="btn"
                type="button"
              >
                7
              </button>
              <button
                class="btn"
                type="button"
              >
                8
              </button>
              <button
                class="btn"
                type="button"
              >
                9
              </button>
              <button
                class="btn"
                type="button"
              >
                x
              </button>
            </div>
            <div
              class="row"
            >
              <button
                class="btn"
                type="button"
              >
                4
              </button>
              <button
                class="btn"
                type="button"
              >
                5
              </button>
              <button
                class="btn"
                type="button"
              >
                6
              </button>
              <button
                class="btn"
                type="button"
              >
                -
              </button>
            </div>
            <div
              class="row"
            >
              <button
                class="btn"
                type="button"
              >
                1
              </button>
              <button
                class="btn"
                type="button"
              >
                2
              </button>
              <button
                class="btn"
                type="button"
              >
                3
              </button>
              <button
                class="btn"
                type="button"
              >
                +
              </button>
            </div>
            <div
              class="row last-row"
            >
              <button
                class="btn"
                type="button"
              >
                0
              </button>
              <button
                class="btn"
                type="button"
              >
                .
              </button>
              <button
                class="btn"
                type="button"
              >
                =
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="contentContainer"
    >
      <h2>
        Let's do some calculations!
      </h2>
      <div
        class="calculator-body"
      >
        <div
          class="screen"
          data-testid="result"
        >
          0
        </div>
        <div
          class="keyboard"
        >
          <div
            class="row"
          >
            <button
              class="btn"
              type="button"
            >
              AC
            </button>
            <button
              class="btn"
              type="button"
            >
              +/-
            </button>
            <button
              class="btn"
              type="button"
            >
              %
            </button>
            <button
              class="btn"
              type="button"
            >
              รท
            </button>
          </div>
          <div
            class="row"
          >
            <button
              class="btn"
              type="button"
            >
              7
            </button>
            <button
              class="btn"
              type="button"
            >
              8
            </button>
            <button
              class="btn"
              type="button"
            >
              9
            </button>
            <button
              class="btn"
              type="button"
            >
              x
            </button>
          </div>
          <div
            class="row"
          >
            <button
              class="btn"
              type="button"
            >
              4
            </button>
            <button
              class="btn"
              type="button"
            >
              5
            </button>
            <button
              class="btn"
              type="button"
            >
              6
            </button>
            <button
              class="btn"
              type="button"
            >
              -
            </button>
          </div>
          <div
            class="row"
          >
            <button
              class="btn"
              type="button"
            >
              1
            </button>
            <button
              class="btn"
              type="button"
            >
              2
            </button>
            <button
              class="btn"
              type="button"
            >
              3
            </button>
            <button
              class="btn"
              type="button"
            >
              +
            </button>
          </div>
          <div
            class="row last-row"
          >
            <button
              class="btn"
              type="button"
            >
              0
            </button>
            <button
              class="btn"
              type="button"
            >
              .
            </button>
            <button
              class="btn"
              type="button"
            >
              =
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Test components Home snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="contentContainer"
      >
        <h1>
          Welcome to our page!
        </h1>
        <div
          class="paragraphs"
        >
          <p>
            A calculator is a handy tool for quickly performing basic mathematical calculations. It can save time and effort compared to doing calculations by hand and also reduce the potential for errors. Basic calculators can perform basic mathematical operations such as addition, subtraction, multiplication and division. They are widely used in many fields such as everyday life, business, and education. A basic calculator is not just a tool for performing mathematical calculations, it is also a tool for simplifying everyday life. It can help with budgeting and managing finances, assist with shopping and comparing prices, and can even be used for cooking and recipe conversions.
          </p>
          <p>
            It is a device that can be found in nearly every household and office, and it has become a necessary accessory for many people. With the advancement of technology, many calculators now have additional features such as memory and percentage functions, making them even more versatile and useful. It's not just a tool but a companion which can make your day to day calculation easier and accurate.
          </p>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="contentContainer"
    >
      <h1>
        Welcome to our page!
      </h1>
      <div
        class="paragraphs"
      >
        <p>
          A calculator is a handy tool for quickly performing basic mathematical calculations. It can save time and effort compared to doing calculations by hand and also reduce the potential for errors. Basic calculators can perform basic mathematical operations such as addition, subtraction, multiplication and division. They are widely used in many fields such as everyday life, business, and education. A basic calculator is not just a tool for performing mathematical calculations, it is also a tool for simplifying everyday life. It can help with budgeting and managing finances, assist with shopping and comparing prices, and can even be used for cooking and recipe conversions.
        </p>
        <p>
          It is a device that can be found in nearly every household and office, and it has become a necessary accessory for many people. With the advancement of technology, many calculators now have additional features such as memory and percentage functions, making them even more versatile and useful. It's not just a tool but a companion which can make your day to day calculation easier and accurate.
        </p>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Test components Quote snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <main>
        <div
          class="contentContainer"
        >
          <p>
            Loading...
          </p>
        </div>
      </main>
    </div>
  </body>,
  "container": <div>
    <main>
      <div
        class="contentContainer"
      >
        <p>
          Loading...
        </p>
      </div>
    </main>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Test components Snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <main>
        <div
          class="contentContainer"
        >
          <p>
            Loading...
          </p>
        </div>
      </main>
    </div>
  </body>,
  "container": <div>
    <main>
      <div
        class="contentContainer"
      >
        <p>
          Loading...
        </p>
      </div>
    </main>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Test components Snapshot for Calculator 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="contentContainer"
      >
        <h2>
          Let's do some calculations!
        </h2>
        <div
          class="calculator-body"
        >
          <div
            class="screen"
            data-testid="result"
          >
            0
          </div>
          <div
            class="keyboard"
          >
            <div
              class="row"
            >
              <button
                class="btn"
                type="button"
              >
                AC
              </button>
              <button
                class="btn"
                type="button"
              >
                +/-
              </button>
              <button
                class="btn"
                type="button"
              >
                %
              </button>
              <button
                class="btn"
                type="button"
              >
                รท
              </button>
            </div>
            <div
              class="row"
            >
              <button
                class="btn"
                type="button"
              >
                7
              </button>
              <button
                class="btn"
                type="button"
              >
                8
              </button>
              <button
                class="btn"
                type="button"
              >
                9
              </button>
              <button
                class="btn"
                type="button"
              >
                x
              </button>
            </div>
            <div
              class="row"
            >
              <button
                class="btn"
                type="button"
              >
                4
              </button>
              <button
                class="btn"
                type="button"
              >
                5
              </button>
              <button
                class="btn"
                type="button"
              >
                6
              </button>
              <button
                class="btn"
                type="button"
              >
                -
              </button>
            </div>
            <div
              class="row"
            >
              <button
                class="btn"
                type="button"
              >
                1
              </button>
              <button
                class="btn"
                type="button"
              >
                2
              </button>
              <button
                class="btn"
                type="button"
              >
                3
              </button>
              <button
                class="btn"
                type="button"
              >
                +
              </button>
            </div>
            <div
              class="row last-row"
            >
              <button
                class="btn"
                type="button"
              >
                0
              </button>
              <button
                class="btn"
                type="button"
              >
                .
              </button>
              <button
                class="btn"
                type="button"
              >
                =
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="contentContainer"
    >
      <h2>
        Let's do some calculations!
      </h2>
      <div
        class="calculator-body"
      >
        <div
          class="screen"
          data-testid="result"
        >
          0
        </div>
        <div
          class="keyboard"
        >
          <div
            class="row"
          >
            <button
              class="btn"
              type="button"
            >
              AC
            </button>
            <button
              class="btn"
              type="button"
            >
              +/-
            </button>
            <button
              class="btn"
              type="button"
            >
              %
            </button>
            <button
              class="btn"
              type="button"
            >
              รท
            </button>
          </div>
          <div
            class="row"
          >
            <button
              class="btn"
              type="button"
            >
              7
            </button>
            <button
              class="btn"
              type="button"
            >
              8
            </button>
            <button
              class="btn"
              type="button"
            >
              9
            </button>
            <button
              class="btn"
              type="button"
            >
              x
            </button>
          </div>
          <div
            class="row"
          >
            <button
              class="btn"
              type="button"
            >
              4
            </button>
            <button
              class="btn"
              type="button"
            >
              5
            </button>
            <button
              class="btn"
              type="button"
            >
              6
            </button>
            <button
              class="btn"
              type="button"
            >
              -
            </button>
          </div>
          <div
            class="row"
          >
            <button
              class="btn"
              type="button"
            >
              1
            </button>
            <button
              class="btn"
              type="button"
            >
              2
            </button>
            <button
              class="btn"
              type="button"
            >
              3
            </button>
            <button
              class="btn"
              type="button"
            >
              +
            </button>
          </div>
          <div
            class="row last-row"
          >
            <button
              class="btn"
              type="button"
            >
              0
            </button>
            <button
              class="btn"
              type="button"
            >
              .
            </button>
            <button
              class="btn"
              type="button"
            >
              =
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Test components Snapshot for Home  1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="contentContainer"
      >
        <h1>
          Welcome to our page!
        </h1>
        <div
          class="paragraphs"
        >
          <p>
            A calculator is a handy tool for quickly performing basic mathematical calculations. It can save time and effort compared to doing calculations by hand and also reduce the potential for errors. Basic calculators can perform basic mathematical operations such as addition, subtraction, multiplication and division. They are widely used in many fields such as everyday life, business, and education. A basic calculator is not just a tool for performing mathematical calculations, it is also a tool for simplifying everyday life. It can help with budgeting and managing finances, assist with shopping and comparing prices, and can even be used for cooking and recipe conversions.
          </p>
          <p>
            It is a device that can be found in nearly every household and office, and it has become a necessary accessory for many people. With the advancement of technology, many calculators now have additional features such as memory and percentage functions, making them even more versatile and useful. It's not just a tool but a companion which can make your day to day calculation easier and accurate.
          </p>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="contentContainer"
    >
      <h1>
        Welcome to our page!
      </h1>
      <div
        class="paragraphs"
      >
        <p>
          A calculator is a handy tool for quickly performing basic mathematical calculations. It can save time and effort compared to doing calculations by hand and also reduce the potential for errors. Basic calculators can perform basic mathematical operations such as addition, subtraction, multiplication and division. They are widely used in many fields such as everyday life, business, and education. A basic calculator is not just a tool for performing mathematical calculations, it is also a tool for simplifying everyday life. It can help with budgeting and managing finances, assist with shopping and comparing prices, and can even be used for cooking and recipe conversions.
        </p>
        <p>
          It is a device that can be found in nearly every household and office, and it has become a necessary accessory for many people. With the advancement of technology, many calculators now have additional features such as memory and percentage functions, making them even more versatile and useful. It's not just a tool but a companion which can make your day to day calculation easier and accurate.
        </p>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Test components Snapshot for Quote 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <main>
        <div
          class="contentContainer"
        >
          <p>
            Loading...
          </p>
        </div>
      </main>
    </div>
  </body>,
  "container": <div>
    <main>
      <div
        class="contentContainer"
      >
        <p>
          Loading...
        </p>
      </div>
    </main>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Test components snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="contentContainer"
      >
        <h1>
          Welcome to our page!
        </h1>
        <div
          class="paragraphs"
        >
          <p>
            A calculator is a handy tool for quickly performing basic mathematical calculations. It can save time and effort compared to doing calculations by hand and also reduce the potential for errors. Basic calculators can perform basic mathematical operations such as addition, subtraction, multiplication and division. They are widely used in many fields such as everyday life, business, and education. A basic calculator is not just a tool for performing mathematical calculations, it is also a tool for simplifying everyday life. It can help with budgeting and managing finances, assist with shopping and comparing prices, and can even be used for cooking and recipe conversions.
          </p>
          <p>
            It is a device that can be found in nearly every household and office, and it has become a necessary accessory for many people. With the advancement of technology, many calculators now have additional features such as memory and percentage functions, making them even more versatile and useful. It's not just a tool but a companion which can make your day to day calculation easier and accurate.
          </p>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="contentContainer"
    >
      <h1>
        Welcome to our page!
      </h1>
      <div
        class="paragraphs"
      >
        <p>
          A calculator is a handy tool for quickly performing basic mathematical calculations. It can save time and effort compared to doing calculations by hand and also reduce the potential for errors. Basic calculators can perform basic mathematical operations such as addition, subtraction, multiplication and division. They are widely used in many fields such as everyday life, business, and education. A basic calculator is not just a tool for performing mathematical calculations, it is also a tool for simplifying everyday life. It can help with budgeting and managing finances, assist with shopping and comparing prices, and can even be used for cooking and recipe conversions.
        </p>
        <p>
          It is a device that can be found in nearly every household and office, and it has become a necessary accessory for many people. With the advancement of technology, many calculators now have additional features such as memory and percentage functions, making them even more versatile and useful. It's not just a tool but a companion which can make your day to day calculation easier and accurate.
        </p>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
